src/css/partials/_base.scss

```diff
@@ -1,6 +1,6 @@
 /* _colors.scss */
 /* unique color names */
-$dodger-blue: #22A5FF;
+$dodger-blue: #2297FF;
```

Comment:
I hope whoever asked you to update this understands it will impact all apps.

Reply:
I'll ask Bret and Josh about it. Bret pointed out the differences in color in some of the common controls like the navigationWizard and the checkboxes, so I checked the mockups for the accounts wizard and the app exchange submission and noticed that the color referred as primary in the CSS of the common controls was `#2297FF` in the mockups, instead of `#22A5FF`. So I decided to update it as it seemed that it was the new primary color and the difference to the old one was slight. I also wanted to know your opinion about it when you reviewed this PR.

![image](https://user-images.githubusercontent.com/893877/65254984-549f2980-daba-11e9-916b-5c65ea0337aa.png)

I made the color change in a single commit in case it needs to be reverted.

Reply:
I asked to Bret and Josh, and they agreed :+1: `#22A5FF` was the old color, and `#2297FF` is the current primary color.

Reply:
Okay, as long as they understand the whole UI will be impacted, even older mockups that where not designed specifically with that color).